### PR TITLE
Workspace video

### PR DESCRIPTION
**Pull request by raabrp**
Merging on github rather than locally. Problems with adding the ``json`` database files to build were confusing me with git.

### CODE REVIEW

File: src/components/workspace.js [hunk 1]
```diff
@@ -502,6 +507,30 @@ class WorkspaceContent extends React.Component {
...
+                    ]
+                ) */
+                let videoTexture = fxChain(this.drawCommands,
+                    [
+                        {name: 'barrelDistort',  buffer: this.barrelBuffer, uniforms: { texture: this.videoTexture, lens: [l.a, l.b, l.F, l.scale], fov: [f.x, f.y] } },
+                        {name: 'image', buffer: null, uniforms: {perspective: this.camera.perspective, view: this.camera.view, location: [0,0,0], size: [workspaceSize.width, workspaceSize.height], selected: false}}  // DRAWS THE RESULT BUFFER ONTO IMAGE
+                    ]
+                )
+                
+            }
```
Comment (sdesalas) [inline src/components/workspace.js]: ☝️  .... 👍 👍

Comment (sdesalas) [inline src/components/workspace.js]: Thanks for merging here. I'll try to get it working locally.

Comment (raabrp) [inline src/components/workspace.js]: Sorry for the late reply. I'm meaning to get it working locally on my branch before pushing it back towards LW4's fork (the ``workspace_video`` branch), letting it get pulled into ``dev-es6`` for it to be upstream of Darkly's fork. Is that not congruent with your thinking?

Comment (raabrp) [inline src/components/workspace.js]: https://github.com/LaserWeb/LaserWeb4/pull/265

**Pull request merged**